sdk/storage/storage-blob/README.md

```diff
@@ -321,40 +321,12 @@ A complete example of basic scenarios is at [samples/basic.ts](https://github.co
 
 ## Troubleshooting
 
-It could help diagnozing issues by turning on the console logging. Here's an example logger implementation. First, add a custom logger:
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
 
 ```javascript
-class ConsoleHttpPipelineLogger {
```

Comment:
Samples need to be fixed too.
https://github.com/Azure/azure-sdk-for-js/blob/81a6e7678f1d074e81d5d98da99fdd3276b67fc6/sdk/storage/storage-blob/samples/typescript/advanced.ts#L13-L19

Can be done in a followup PR.

Reply:
Good call!